command.py

```diff
@@ -66,15 +66,15 @@
 }
 
 multipart_command_types = {
-    'new_waypoint':             {'cmd':128, 'type':'ddffB'},
-    'insert_waypoint':          {'cmd':129, 'type':'ddffBBB'},
-    'set_returnHomeCoordinates':{'cmd':130, 'type':'ddf'},
-    'tare_IMU':                 {'cmd':131,  'type':'fff'},
-    'set_IMU':                  {'cmd':132,  'type':'fff'},
-    'set_KDValues':              {'cmd':133,  'type':'fffffff'},
-    'set_KPValues':              {'cmd':134,  'type':'fffffff'},
-    'set_KIValues':              {'cmd':135,  'type':'fffffff'},
-    'update_waypoint':          {'cmd':136, 'type':'ddffBB'},
+    'new_waypoint':             {'cmd':128, 'type':'ddffB'}, # lon, lat, alt, rad, type
```

Comment:
I am not convinced it will work without it except by accident. If you don't have the padding, the upload packet will be the wrong size and the pic pilot would overrun when trying to copy out.